Source/WebCore/accessibility/atspi/AccessibilityObjectTextAtspi.cpp

```diff
@@ -787,7 +787,7 @@ AccessibilityObjectAtspi::TextAttributes AccessibilityObjectAtspi::textAttribute
         addAttributeIfNeeded("family-name"_s, style.fontCascade().firstFamily());
         addAttributeIfNeeded("size"_s, makeString(std::round(style.computedFontSize() * 72 / WebCore::fontDPI()), "pt"_s));
         addAttributeIfNeeded("weight"_s, makeString(static_cast<float>(style.fontCascade().weight())));
-        addAttributeIfNeeded("style"_s, style.fontCascade().italic() ? "italic"_s : "normal"_s);
+        addAttributeIfNeeded("style"_s, style.fontCascade().fontStyleSlope() ? "italic"_s : "normal"_s);
```

Comment:
Why doesn't this use a function named `isConsideredItalic`? I'm sure there is a subtle distinction between how the cascade expresses this and how she style does, but I don't fully appreciate it.

Reply:
I think some of these probably pre-date some of the additional complexity of font-style.